samsawan/SendBird-JavaScript
# widget no longer relies on window for functionality

# Things Done
- no longer attach chat widget to windows object
  - instead, export the `SBWidget` class and use that with great aplomb
- no longer expose the minified JSON
  - front end bloat is, apparently, solved by caching
- minor formatting changes to reduce number of IDE warnings

# How To Test
- no new functionality
- `npm start`
  - ensure the widget is still there and functional
  - ensure the widget is _not on_ the windows object
